app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -234,42 +223,6 @@ def update_contention_classification_all!
     classifier_response['is_fully_classified']
   end
 
-  # Submits contention information to the VRO contention classification
-  # service for single-contention claims.
-  #
-  # note: this method is only used for single-contention claims and is
-  # deprecated in favor of update_contention_classification_all, which handles
-  # both single and multi-contention claims
-  def update_contention_classification_single_contention!
```

Comment:
Thanks for catching these, I removed these in the latest commit. For removing the names in the `config/features.yml` , [platform docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide) say to remove these after the changes to remove references are deployed to prod. Do you know if this order matters that much or if it is safe to remove it all? I imagine it is safe because the code is not referencing these anymore, but just wanted to make sure based on the documentation.

Reply:
@tyler-spangler6 I can answer that. After deploying these changes to production, you can open another PR to remove the features. The order matter because there's a different deployment process for the features. 

After the `features.yml` PR is deployed, You can open a support ticket to remove those features from database.